frontend/vite.config.js

```diff
@@ -6,6 +6,7 @@ import vue from '@vitejs/plugin-vue'
 // https://vitejs.dev/config/
 export default defineConfig(({command, mode}) => {
     const env = loadEnv(mode, process.cwd(), '')
+    console.log(env.SERVER_PORT)
```

Comment:
Console log.. do we want to keep this ?

Reply:
Sorry, I forgot to delete this back again